src/redux/message/message.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+
+const FETCH_STARTED = 'hello-rails-react/app/javascript/redux/message/FETCH_STARTED';
+const FETCH_SUCCEDED = 'hello-rails-react/app/javascript/redux/message/FETCH_SUCCEDED';
+const FETCH_FAILED = 'hello-rails-react/app/javascript/redux/message/FETCH_FAILED';
+const MESSAGES_PATH = 'https://shielded-waters-51364.herokuapp.com/api/v1/random-greeting';
```

Comment:
- [ ] I get random greetings from Heroku but the problem is I can't test locally these two apps to see if they work together because I can't create a database by running 'rails db: create` I always get errors and I am not able to run the server as well, please check your database setup and test it yourself before submitting again test your app by using local host